[2023-12-27 16:30:04,970] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.cargar_data manual__2023-12-27T16:19:37.576458+00:00 [queued]>
[2023-12-27 16:30:05,016] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: OpenWeather_ETL2.cargar_data manual__2023-12-27T16:19:37.576458+00:00 [queued]>
[2023-12-27 16:30:05,017] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 16:30:05,020] {taskinstance.py:1377} INFO - Starting attempt 3 of 6
[2023-12-27 16:30:05,021] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 16:30:05,092] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): cargar_data> on 2023-12-27 16:19:37.576458+00:00
[2023-12-27 16:30:05,103] {standard_task_runner.py:52} INFO - Started process 7775 to run task
[2023-12-27 16:30:05,110] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'OpenWeather_ETL2', 'cargar_data', 'manual__2023-12-27T16:19:37.576458+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/ETL_OpenWeather2.py', '--cfg-path', '/tmp/tmpxr3sul3_', '--error-file', '/tmp/tmp3p80clih']
[2023-12-27 16:30:05,118] {standard_task_runner.py:80} INFO - Job 95: Subtask cargar_data
[2023-12-27 16:30:05,592] {task_command.py:371} INFO - Running <TaskInstance: OpenWeather_ETL2.cargar_data manual__2023-12-27T16:19:37.576458+00:00 [running]> on host 44a36bef83ad
[2023-12-27 16:30:05,945] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-27 16:30:06,225] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Furtado_Julio
AIRFLOW_CTX_DAG_ID=OpenWeather_ETL2
AIRFLOW_CTX_TASK_ID=cargar_data
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T16:19:37.576458+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-27T16:19:37.576458+00:00
[2023-12-27 16:30:06,230] {logging_mixin.py:115} INFO - Cargando la data para la fecha: 2023-12-27 16
[2023-12-27 16:30:06,325] {logging_mixin.py:115} INFO - (1, 18)
[2023-12-27 16:30:06,453] {logging_mixin.py:115} INFO -                     dt          name  ...         sunrise_time          sunset_time
0  2023-12-27 13:19:44  Buenos Aires  ...  2023-12-27 05:40:26  2023-12-27 20:08:14

[1 rows x 18 columns]
[2023-12-27 16:30:08,607] {logging_mixin.py:115} INFO - Carga completada en Redshift!
[2023-12-27 16:30:08,609] {python.py:173} INFO - Done. Returned value was: None
[2023-12-27 16:30:08,657] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=OpenWeather_ETL2, task_id=cargar_data, execution_date=20231227T161937, start_date=20231227T163004, end_date=20231227T163008
[2023-12-27 16:30:08,711] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-27 16:30:08,796] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
